eclipse-theia/theia
# Os/fix command new window

#### What it does

'New Window' command added for the electron and the browser targets. The keybindings and the menu registration of the command are also supported for both of them. 

#### How to test

1. Open a project (or a file) in a new workspace
2. You can choose between these actions to spawn a new window :
   - Entering CTRL + SHIFT + N (electron target) or CTRL + ALT + N (browser target)
   - Clicking on the 'New Window' option in the 'File' menu
   - Searching and clicking on the 'New Window' option after entering the command palette (CTRL + P) in the 'View' menu
3. Check that the new window displays the welcome page and that no workspace has been opened with it

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
